vm/src/obj/objstr.rs

```diff
@@ -532,6 +532,22 @@ impl PyString {
         )
     }
 
+    #[pymethod]
+    fn removeprefix(&self, pref: PyStringRef) -> PyResult<String> {
+        if self.value.as_str().starts_with(&pref.value) {
+            return Ok(self.value[pref.len()..].to_string());
+        }
+        Ok(self.value.to_string())
+    }
+
+    #[pymethod]
+    fn removesuffix(&self, suff: PyStringRef) -> PyResult<String> {
+        if self.value.as_str().ends_with(&suff.value) {
+            return Ok(self.value[..self.value.len() - suff.len()].to_string());
```

Comment:
`self.value.len()` is a rust string length and `suff.len()` is a python string length. These numbers should not be mixed because they heve different value for non-ascii string. Will you please add non-ascii tests to the test code?

Reply:
Thanks, pretty obvious fault.

Reply:
in the py39  tests I found no such tests when creating it. maybe I  missed it or the test are also behind. anyhow as you are not interested at the moment I ll stop sening prs.